packages/contracts/contracts/CollateralManager.sol

```diff
@@ -25,6 +25,13 @@ contract CollateralManager is OwnableUpgradeable, ICollateralManager {
     mapping(uint256 => address) public _escrows; // bidIds -> collateralEscrow
     // bidIds -> validated collateral info
     mapping(uint256 => CollateralInfo) internal _bidCollaterals;
+
+    /*
+        Since collateralInfo is mapped (address assetAddress => Collateral) that means
+        that only a single tokenId per nft per loan can be collateralized. 
+         Ex. Two bored apes cannot be used as collateral for a single loan.
+
+    */
```

Comment:
```suggestion
    /*
     * Since collateralInfo is mapped (address assetAddress => Collateral) that means
     * that only a single tokenId per nft per loan can be collateralized. 
     * Ex. Two bored apes cannot be used as collateral for a single loan.
     */
```